docs/content/core/volumes.md

```diff
@@ -57,7 +57,25 @@ and errors with Docker Desktop on macOS. See [docksal/docksal#678](https://githu
 
 `DOCKSAL_VOLUMES` value changes the mount type of the project volumes mentioned above and also affects `fin` behavior. 
 
-### bind 
+The value can be set globally (all projects)
+
+```bash
+fin config set --global DOCKSAL_VOLUMES=nfs
```

Comment:
Did you mean to have DOCKSAL…VOLUMES=bind here?

Reply:
I used different values as examples. Maybe I should keep them consistent or use `<value>` as a placeholder instead.

Reply:
I asked because all the examples and this section title were bind.